core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -199,26 +198,79 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
     s"$org/$name"
   }
 
+  val kafka = new AtomicReference[KafkaContainer]()
+
+  def setupKafka(topics: Seq[String])(implicit log: Logger) = {
+
+    val cl = Thread.currentThread().getContextClassLoader()
+    val kafkaPort =
+      try {
+        val c = getClass().getClassLoader()
+        Thread.currentThread().setContextClassLoader(c)
+
+        val k = new KafkaContainer(tcutility.DockerImageName.parse("confluentinc/cp-kafka:5.4.3"))
+          .withExposedPorts(KafkaPort)
+          .waitingFor(
+            Wait.forLogMessage(".*Kafka startTimeMs.*\\n", 1)
+          )
+        k.start()
+        kafka.set(k)
+
+        k.getMappedPort(KafkaPort)
+      } finally {
+        Thread.currentThread().setContextClassLoader(cl)
+      }
+
+    log.debug(s"Setting up Kafka broker in Docker on port: $kafkaPort")
+
+    import org.apache.kafka.clients.admin.{ AdminClient, AdminClientConfig, NewTopic }
+    import scala.collection.JavaConverters._
+
+    var retry = 5
+
+    while (retry > 0) {
```

Comment:
added retries, since it looks like that a Kafka freshly started is sometimes failing to create the topics.